src/Functions/Public/Connect-vRAServer.ps1

```diff
@@ -12,6 +12,7 @@
         .PARAMETER Username
         Username to connect with
         For domain accounts ensure to specify the Username in the format username@domain, not Domain\Username
+        Note: UPN's are valid login Usernames as well and may also be in the format username@domain
```

Comment:
```suggestion
        Note: UPNs are valid login Usernames as well and may also be in the format username@domain
```

Reply:
Just submitted the fix for this typo in latest commit to this pull request.